definitions/ext-istio_service/definition.yml

```diff
@@ -22,6 +22,88 @@ synthesis:
     conditions:
     - attribute: metricName
       prefix: istio_tcp
+      
+  # Support for prometheus instrumentation provider
+  - identifier: "service_istio_io_canonical_name"
+    encodeIdentifierInGUID: true
+    name: "service_istio_io_canonical_name"
+    conditions:
+    - attribute: metricName
+      prefix: istio_request
+    tags:
+      service_istio_io_canonical_name:
```

Comment:
This is the name of the entity so it shouldn't be a tag (if you can remove it from the global tags that would be great also)

Reply:
The previous revision had the name already as a tag. Shall I still remove it?

Reply:
yes please, that was a mistake we seemed to overlook at the time.